roles/oVirt.vm-infra/tasks/manage_state.yml

```diff
@@ -0,0 +1,46 @@
+---
+########################################################################
+# Legacy modification of cloud_init:
+########################################################################
+- set_fact:
+    cloud_init: "{{ current_vm.cloud_init | default(current_vm.profile.cloud_init) | default({}) }}"
+
+- set_fact:
+    cloud_init: "{{ cloud_init | combine({'user_name': cloud_init.user_name | default('root') }) }}"
+
+- set_fact:
+    cloud_init: "{{ cloud_init | combine({'authorized_ssh_keys': current_vm.ssh_key | default(current_vm.profile.ssh_key)}) }}"
+  when: current_vm.profile.ssh_key is defined or current_vm.ssh_key is defined
+
+- set_fact:
```

Comment:
No need for this here; We require ssh keys on the instances for ansible.

Reply:
@michaelgugino This is how those ssh keys end up on the nodes at deployment time. 
Actually, this whole stanza is a backwards-compatibility exception, so it could be removed.